cmake/installdata/windows/dlls.txt

```diff
@@ -1,3 +1,4 @@
 system/*.dll .
+system/libbluray*.jar .
 project/Win32BuildSetup/dependencies/python27.dll .
 project/BuildDependencies/${ARCH}/bin/*.dll .
```

Comment:
add `project/BuildDependencies/${ARCH}/bin/libbluray*.jar .` to also work for x64